src/tox_current_env/hooks4.py

```diff
@@ -181,13 +181,13 @@ def create_python_env(self):
                 os.symlink(sys.executable, symlink)
 
     def env_bin_dir(self):
-        return Path(sysconfig.get_path("scripts", sysconfig.get_default_scheme()))
+        return Path(get_path("scripts", get_default_scheme()))
```

Comment:
Is it necessary to explicitly call `get_default_scheme()` here? Isn't that what get_paths does by default?